system/handlers/admin/DataHelpers.cfc

```diff
@@ -161,18 +163,20 @@ component extends="preside.system.base.adminHandler" {
 	 *
 	 */
 	public void function getRecordsForRelatedRecordsDatatable( event, rc, prc ) {
-		var objectName     = rc.objectName   ?: "";
-		var propertyName   = rc.propertyName ?: "";
-		var recordId       = rc.recordId     ?: "";
-		var gridFields     = adminDataViewsService.listGridFieldsForRelationshipPropertyTable( objectName, propertyName ).toList();
-		var relatedObject  = presideObjectService.getObjectPropertyAttribute( objectName=objectName, propertyName=propertyName, attributeName="relatedTo" );
-		var relatedIdField = presideObjectService.getIdField( objectName=relatedObject );
-		var extraFilters   = [];
-		var subquerySelect = presideObjectService.selectData(
+		var objectName       = rc.objectName       ?: "";
+		var propertyName     = rc.propertyName     ?: "";
+		var recordId         = rc.recordId         ?: "";
+		var fromVersionTable = rc.fromVersionTable ?: false;
```

Comment:
You can ensure that this variable is a boolean (so no need for BooleanFormat(), below), with:

```
var fromVersionTable = isTrue( rc.fromVersionTable ?: "" );
```

That is the preferred approach here.

Reply:
this is updated.